docs/topics/http/urls.txt

```diff
@@ -661,7 +668,12 @@ included at all).
 
 You may also use the same name for multiple URL patterns if they differ in
 their arguments. In addition to the URL name, :func:`~django.urls.reverse()`
-matches the number of arguments and the names of the keyword arguments.
+matches the number and type of arguments and the names of the keyword arguments.
```

Comment:
` and type of argument ...` `reverse()` doesn't match type of arguments it is sth that path converters do. I will revert this change.